WordPress/Sniffs/Theme/NoTitleTagSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
+/**
+ * WordPress_Sniffs_Theme_NoTitleTagSniff.
+ *
+ * Forbids the use of the <title> tag, unless it is within a <svg> tag.
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    carolinan
+ */
```

Comment:
Please update the class documentation to follow the (new) pattern as introduced in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/651